src/widgets/__tests__/__snapshots__/HospitalizationDeathChart.test.tsx.snap

```diff
@@ -27,21 +27,8 @@ exports[`<HospitalizationDeathChart> (died plot) dataset1 renders correctly 1`]
           className="sc-gTRrQi elYrUj"
         >
           <div
-            className="sc-ewnqHT fpGYCl"
-          >
-            <div
-              className="recharts-responsive-container"
-              style={
-                Object {
-                  "height": "100%",
-                  "maxHeight": undefined,
-                  "minHeight": undefined,
-                  "minWidth": undefined,
-                  "width": "100%",
-                }
-              }
-            />
-          </div>
+            className="sc-ewnqHT bRaosl"
```

Comment:
why have the `recharts-responsive-container`s been removed and why has the ID of the className changed?

Reply:
The IDs are from `styled-components` and I think they changed because I made some changes to the metrics.